### PR TITLE
bump dugite to get reflog fix

### PR DESCRIPTION
**Pull request by shiftkey**
- [x] add regression test to close out #2426 ???

### CODE REVIEW

**Comment by joshaber** [comment]
Tests added 👍

**Comment by joshaber** [comment]
🏷 🐥

**Comment by shiftkey** [comment]
As myself and @joshaber have touched this PR, would you mind doing the needful @iAmWillShepherd?

File: app/test/unit/git/reflog-test.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          null,
+          new Commit('', '', '', new CommitIdentity('', '', new Date()), []),
+          BranchType.Local
+        ),
+        'branch-1-test'
+      )
+
+      const branches = await getRecentBranches(repository!, 10)
+      expect(branches).to.contain('branch-1')
+      expect(branches).to.contain('branch-2')
```
Comment (iAmWillShepherd) [inline app/test/unit/git/reflog-test.ts]: If the branch is renamed, why isn't it expected in the assertions?

Comment (joshaber) [inline app/test/unit/git/reflog-test.ts]: Our recent branches logic doesn't track branches across renames. It'd be nice if it did, but that's a separate issue.

**Comment by joshaber** [comment]
#jolo'ing this so that we can get it in prerelease today and public tomorrow.

**Pull request merged**